### DISCUSSION TITLE
augmentation and regularization used in MLP-Mixer

### DISCUSSION THREAD

**Discussion by Ga-Lee** [google-research/big_vision#71] [Q&A] [answered]
Hello! Thank you for your work!
In the paper of MLP-Mixer, when training mixer-B/16 on imagenet1k from  scratch, it is said extra regularization is applied to gain the  the accuracy of 76% , I want to know what detailed augmentation and regularization strategy  is used for the experiment?  Is there any config file can be found? 
Thank you for your help! : )

**Comment by akolesnikoff** [comment, marked as answer]
Hi!

I've just submitted the MLP-Mixer code, as well as the configuration that achieves 76% top-1 ImageNet with MLP-Mixer-B/16 model: https://github.com/google-research/big_vision/blob/main/big_vision/configs/mlp_mixer_i1k.py. Let me know if you have any further questions.

**Comment by Ga-Lee** [comment]
> Hi!
> 
> I've just submitted the MLP-Mixer code, as well as the configuration that achieves 76% top-1 ImageNet with MLP-Mixer-B/16 model: https://github.com/google-research/big_vision/blob/main/big_vision/configs/mlp_mixer_i1k.py. Let me know if you have any further questions.

Got it !
Thank you for your reply!
